openstack_controller/tests/metrics.py

```diff
@@ -157,6 +157,20 @@
 
 NEUTRON_PROJECT_METRICS = NEUTRON_NETWORK_METRICS | NEUTRON_QUOTA_METRICS
 
+VOLUMES_METRICS_CINDER = [
```

Comment:
this is not used

Reply:
oops the base branch must not have updated. I've removed it.